src/Common/src/Interop/Windows/mincore/Interop.Errors.cs

```diff
@@ -8,27 +8,50 @@ internal partial class mincore
         internal partial class Errors
         {
             internal const int ERROR_SUCCESS = 0x0;
+            internal const int ERROR_INVALID_FUNCTION = 0x1;
             internal const int ERROR_FILE_NOT_FOUND = 0x2;
             internal const int ERROR_PATH_NOT_FOUND = 0x3;
             internal const int ERROR_ACCESS_DENIED = 0x5;
             internal const int ERROR_INVALID_HANDLE = 0x6;
+            internal const int ERROR_NOT_ENOUGH_MEMORY = 0x8;
+            internal const int ERROR_INVALID_DATA = 0xD;
+            internal const int ERROR_INVALID_DRIVE = 0xF;
+            internal const int ERROR_NO_MORE_FILES = 0x12;
+            internal const int ERROR_NOT_READY = 0x15;
+            internal const int ERROR_BAD_LENGTH = 0x18;
             internal const int ERROR_SHARING_VIOLATION = 0x20;
-            internal const int ERROR_NOT_READY = 21;
+            internal const int ERROR_LOCK_VIOLATION = 0x21;
+            internal const int ERROR_HANDLE_EOF = 0x26;
             internal const int ERROR_FILE_EXISTS = 0x50;
             internal const int ERROR_INVALID_PARAMETER = 0x57;
             internal const int ERROR_BROKEN_PIPE = 0x6D;
+            internal const int ERROR_INSUFFICIENT_BUFFER = 0x7A;
+            internal const int ERROR_INVALID_NAME = 0x7B;
+            internal const int ERROR_NEGATIVE_SEEK = 0x83;
+            internal const int ERROR_DIR_NOT_EMPTY = 0x91;
+            internal const int ERROR_BAD_PATHNAME = 0xA1;
             internal const int ERROR_ALREADY_EXISTS = 0xB7;
-            internal const int ERROR_FILENAME_EXCED_RANGE = 0xCE;  // filename too long.
+            internal const int ERROR_ENVVAR_NOT_FOUND = 0xCB;
+            internal const int ERROR_FILENAME_EXCED_RANGE = 0xCE;
+            internal const int ERROR_PIPE_BUSY = 0xE7;
             internal const int ERROR_NO_DATA = 0xE8;
+            internal const int ERROR_PIPE_NOT_CONNECTED = 0xE9;
             internal const int ERROR_MORE_DATA = 0xEA;
             internal const int ERROR_LOCK_FAILED = 167;
             internal const int ERROR_BUSY = 170;
             internal const int ERROR_BAD_EXE_FORMAT = 193;
             internal const int ERROR_EXE_MACHINE_TYPE_MISMATCH = 216;
             internal const int ERROR_PARTIAL_COPY = 299;
```

Comment:
Some of these values are hex and some are decimal.  We should be consistent. It'd also be good to sort the list in some way... right now it doesn't appear to be sorted by either name or value.